pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,198 @@
+---
+$title: Embed stories in web pages
+$order: 3
+description: 'The AMP Story Player enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.'
+formats:
+  - websites
+  - stories 
+---
+
+Stories are a full-screen immersive experience. They’re hosted on the open web with their own URL, making them easily shareable. But, what if you want to integrate stories into your own site, for example, within a blog, product description or news article? 
+
+The [AMP Story Player](https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md) enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.
+
+# Display stories in a non-AMP page
+
+You can embed AMP stories inside a non-AMP page, allowing users to tap or click through the experience without leaving the host document!
+
+[example preview="top-frame" playground="false"]
+```html
+<!doctype html>
+    <head>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+        async
+        src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+        href="https://cdn.ampproject.org/amp-story-player-v0.css"
+        rel="stylesheet"
+        type="text/css"
+    />
+    <style>
+        header {
+        height: 8vh;
+        width: 100vw;
+        display: flex;
+        justify-content: space-around;
+        color: #545454;
+        background-color: #DDB556;
+        margin-bottom: 3vh;
+        }
+        .content {
+        display: flex;
```

Comment:
Use `margin: 1rem auto` instead of a flex layout for centering in this case.

Reply:
This is more to center the viewer than the text:
<img width="1410" alt="Screen Shot 2020-03-19 at 12 11 56 PM" src="https://user-images.githubusercontent.com/23297417/77105412-df65d000-69da-11ea-8a52-22e8b25ae418.png">
<img width="1392" alt="Screen Shot 2020-03-19 at 12 12 05 PM" src="https://user-images.githubusercontent.com/23297417/77105441-e856a180-69da-11ea-8d54-8ff43bcbe5e4.png">

Reply:
Remove the content div as you don't need it. This does the trick:

```
amp-story-player {
  margin: 0 auto;
}
```

Reply:
nvm